networkx/algorithms/tests/test_similarity.py

```diff
@@ -954,6 +954,22 @@ def test_generate_random_paths_weighted(self):
         assert expected_paths == list(paths)
         assert expected_map == index_map
 
+    def test_one_node_one_loop_and_empty_graph(self):
+        g1 = nx.DiGraph()
+        g1.add_nodes_from([0])
+        g1.add_edges_from([(0, 0)])
+        g2 = nx.DiGraph()
+        ged = nx.graph_edit_distance(g1, g2)
+        assert ged == 2
+
+    def test_one_node_two_loops_and_empty_graph(self):
+        g1 = nx.MultiDiGraph()
+        g1.add_nodes_from([0])
+        g1.add_edges_from([(0, 0), (0, 0)])
+        g2 = nx.DiGraph()
+        ged = nx.graph_edit_distance(g1, g2)
+        assert ged == 3
+
```

Comment:
Might as well add the [multi-self-loop test](https://github.com/networkx/networkx/issues/8188#issuecomment-3172835209) as well while we're at it!